index.ts

```diff
@@ -59,7 +60,14 @@ export function load(app: Application) {
 		type: ParameterType.Path,
 	});
 
+	app.options.addDeclaration({
+		name: "coverageOutputType",
+		help: "Defines the type of the coverage file to be written (svg, json, all).",
+		type: ParameterType.String,
+	});
+
 	app.renderer.on(Renderer.EVENT_END, (event: RendererEvent) => {
+		const notDocumented = [];
```

Comment:
Declare the type please, not doing so has TypeScript treat this as `any[]`
```suggestion
		const notDocumented: string[] = [];
```